core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -675,6 +675,11 @@
       "The maximum number of threads to use to sort logs during" + " recovery", "1.5.0"),
   TSERV_SORT_BUFFER_SIZE("tserver.sort.buffer.size", "10%", PropertyType.MEMORY,
       "The amount of memory to use when sorting logs during recovery.", "1.5.0"),
+  TSERV_WAL_SORT_FILE_PREFIX("tserver.wal.sort.file.", null, PropertyType.PREFIX,
+      "The rfile properties to use when sorting logs during recovery. Most of the properties"
+          + " that begin with 'table.file' can be used here. For example, to set the compression"
```

Comment:
Do you know of 'table.file' props that can't be used here?

Reply:
Yeah but I think they will just get ignored. Properties like "table.file.max" or "table.file.summary.maxSize" that don't apply to these files.